src/main/scala/slamdata/SbtSlamData.scala

```diff
@@ -273,6 +287,70 @@ object SbtSlamData extends AutoPlugin {
   private def transferScripts(baseDir: File, srcs: String*) =
     srcs.foreach(src => transfer(src, baseDir / "scripts" / src, Set(OWNER_EXECUTE)))
 
+  private def slamdataVersionCheck = Def.task {
+    import scala.sys.process._
+
+    val logger: Logger = streams.value.log
+    def relog(line: String): Unit = {
+      val level = """\[([^]]*)\].*""".r
```

Comment:
Should have a leading `^` to ensure that it only matches at start of line. That should also incidentally improve the performance.

Reply:
By default a regex match in Scala is a full string match, which is why the `.*` is needed. But will do.